<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IndirectFitPreviewPlot</class>
 <widget class="QWidget" name="IndirectFitPreviewPlot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>663</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QComboBox" name="cbDataSelection"/>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="loPlotOptions">
     <item>
      <widget class="QLabel" name="lbPlotSpectrum">
       <property name="text">
        <string>Plot Spectrum:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="swPlotSpectrum">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="pgPlotSpinBox">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QSpinBox" name="spPlotSpectrum"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pgPlotCombo">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QComboBox" name="cbPlotSpectrum"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <spacer name="hspFitSingle">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbFitSingle">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Fit Current Preview</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hspPlotPreview">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbPlotPreview">
       <property name="text">
        <string>Plot Current Preview</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hspPlotGuess">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="ckPlotGuess">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Plot Guess</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
